src/templates/homepage/HeroSection/HeroImageOnlyLayout.tsx

```diff
@@ -0,0 +1,27 @@
+import editorStyles from "styles/isomer-cms/pages/Editor.module.scss"
+
+import { getClassNames } from "templates/utils/stylingUtils"
+
+export const HeroImageOnlyLayout = () => {
+  return (
+    <div className={getClassNames(editorStyles, ["bp-hero-body"])}>
+      <div
+        className={getClassNames(editorStyles, [
+          "bp-container",
+          "margin--top--lg",
+        ])}
+      >
+        <div
+          className={getClassNames(editorStyles, [
+            "row",
+            "is-vcentered",
+            "is-centered",
+            "ma",
+          ])}
+        >
+          <div className={getClassNames(editorStyles, ["min-height-mobile"])} />
```

Comment:
Should this tag be closed as `<div></div>` instead of being a self-closing tag? `div` is not a [void element](https://html.spec.whatwg.org/multipage/syntax.html#void-elements), which may cause rendering issues.

Reply:
on auto save, `<div> </div` auto formats to `<div />` ._. it seems like the two should be equivalent?

Reply:
Ah okay, I think not super important.